wgsl/index.bs

```diff
@@ -3730,25 +3728,26 @@ as an entry point.
 When configuring the stage in the pipeline, the entry point is specified by providing
 the [SHORTNAME] module and the entry point's function name.
 
-An entry point function must have no parameters,
-and its return type must be [=void=].
+The parameters of an entry point have to be within [=Entry point IO type=]s.
+They have [=storage classes/in=] storage class.
+The return type of an entry point has to be of an [=Entry point IO type=], or [=void=].
 
 <div class='example wgsl global-scope' heading='Entry Point'>
   <xmp>
-    [[builtin(position)]]   var<out> gl_Position  : vec4<f32>;
-    [[builtin(frag_coord)]] var<in> coord_in : vec4<f32>;
-    [[location(0)]] var<out> color : vec4<f32>;
-
     [[stage(vertex)]]
-    fn vtx_main() -> void { gl_Position = vec4<f32>(); }
-       // OpEntryPoint Vertex %vtx_main "vtx_main" %gl_Position
+    fn vtx_main() -> [[builtin(position)]] vec4<f32> {
+      return vec4<f32>(0.0, 0.0, 0.0, 1.0);
+    }
+       // OpEntryPoint Vertex %vtx_main "vtx_main" %(gl_Position)
```

Comment:
There is no `gl_Position` anymore, so what do we put as the entry point value?

Reply:
It shouldn't matter for the spec, really. In SPIR-V, this is just an "id". We just need to tell the reader that it's meant to be our result value.

Reply:
It maybe better to spell out in the spec that that `vec4` in the return is actually turned into an `OpVariable` with some set of decorations which is what then goes into the `OpEntryPoint` call